### DISCUSSION TITLE
[color_scheme]lock color in standalone charts and allow user to set custom default color for all charts

### DISCUSSION THREAD

**Discussion by junlincc** [apache/superset#18505] [Ideas]
Currently, when user opens a chart that 1)has not been added to any dashboard yet 2)color scheme is selected to the first in dropdown by default, the chart appears in random color sometimes when user views it. which can be confusing.
another related case is that users often choose the same color scheme for most/all charts within their organization, it will be very helpful if we allow user select a default color scheme once and apply to all charts. 

<img width="382" alt="Screen Shot 2021-09-01 at 2 29 50 PM" src="https://user-images.githubusercontent.com/67837651/131747870-3573b57c-08c1-44cb-8648-5ec4460cd6b1.png">

Proposed solution : may be add a switch next to the select checkmark to set selected color by default for all charts(?) or in system setting (?) 

Please note that, this change should not affect the existing behavior in dashboard - color scheme selection in dashboard take precedence over. There are some other follow up tasks in this project....

**Comment by lilila** [comment]
It would be great to create custom color palette from this interface (instead of having to declare them in config.py). 
The color schemes could be ideally stored in the backend database, thus it would be possible to keep the palette when exporting/importing charts . 
The best would still be to have label colors working in Explore and Standalone mode but you mentioned it is not planned soon :/

**Comment by amitmiran137** [comment]
> It would be great to create custom color palette from this interface (instead of having to declare them in config.py). 
> The color schemes could be ideally stored in the backend database, thus it would be possible to keep the palette when exporting/importing charts . 
> The best would still be to have label colors working in Explore and Standalone mode but you mentioned it is not planned soon :/

What you're implying is basically a color scheme editor that manage color schemes against flask web server and persist data
If you're willing to develop I'll help you push this forward

**Reply by lilila** [reply]
Thank you but I do not have time to do this. 
Note: I noted label Colors are back in standalone mode in 1,4 (still not in standalone unfortunately) 
But that would be so great to edit palettes and be able to assign a mapping between the colors and the labels 🤩

I like to always represent the same data with the same colours across all dashboards (I have colour scale for ages, or another for gender, country for instance). I think it really helps when you navigate in the dashboard collection to identify data

**Reply by SharonCastel** [reply]
We also need this feature, it just confuses the customers when the dashboard has different colors for the same values.
Especially when we use the dashboard filter, them the values change and also the colors.
We tried to put only 2 colors in the json but the dashboard used additional colors.
